### PR TITLE
feat: v1 cluster upgrade entrance

### PR DESCRIPTION
**Pull request by xuzhu-591**

### CODE REVIEW

File: src/locales/en-US/pages.ts [hunk 1]
```diff
@@ -609,6 +610,10 @@ export default {
   'pages.message.cluster.deployAll.content4': 'Note that: ',
   'pages.message.cluster.deployAll.content5': '1. If the number of instances is too large, please pay attention to the pressure on the environment',
   'pages.message.cluster.deployAll.content6': '2. In addition to the pre and online environments, the maximum number of replicas in the deploy process is 200% for efficiency.',
+  'pages.message.cluster.upgrade.tooltip': 'Upgrade to version 2.0 for new features such as JDK17 and generalized templates',
```
Comment (kilosonc) [inline src/locales/en-US/pages.ts]: generalized?

File: src/pages/clusters/Detail/Basic/index.tsx [hunk 2]
```diff
@@ -111,6 +112,21 @@ export default function Basic(props: any) {
...
 
+  const onClickUpgrade = () => {
+    Modal.confirm({
+      title: intl.formatMessage({ id: 'pages.message.cluster.upgrade.confirm' }),
+      content: intl.formatMessage({ id: 'pages.message.cluster.upgrade.content' }),
+      onOk() {
+        upgradeCluster(cluster.id).then(() => {
+          successAlert(intl.formatMessage({ id: 'pages.message.cluster.upgrade.success' }));
+          setTimeout(() => {
+            location.reload();
```
Comment (kilosonc) [inline src/pages/clusters/Detail/Basic/index.tsx]: history.push?

**Review by kilosonc — CHANGES_REQUESTED** [review]

**Comment by kilosonc** [comment]
Where is the relative backend PR?

File: src/pages/clusters/Detail/Basic/index.tsx [hunk 3]
```diff
@@ -146,6 +162,19 @@ export default function Basic(props: any) {
         >
           {intl.formatMessage({ id: 'pages.clusterDetail.basic.edit' })}
         </Button>
+        {
+          version === pipelineV2 || (
+            <Tooltip title={intl.formatMessage({ id: 'pages.message.cluster.upgrade.tooltip' })}>
+              <Button
```
Comment (kilosonc) [inline src/pages/clusters/Detail/Basic/index.tsx]: Why not using microapp? It's a button for internal usage.

Comment (xuzhu-591) [reply src/pages/clusters/Detail/Basic/index.tsx]: fixed

File: src/locales/en-US/pages.ts (hunk 1 above)
Comment (xuzhu-591) [reply src/locales/en-US/pages.ts]: => powerful

File: src/pages/clusters/Detail/Basic/index.tsx (hunk 2 above)
Comment (xuzhu-591) [reply src/pages/clusters/Detail/Basic/index.tsx]: I need refresh page.

**Review by ryanwuer — APPROVED** [review]

**Review by kilosonc — APPROVED** [review]

**Review by ryanwuer — APPROVED** [review]

**Review by ryanwuer — APPROVED** [review]

**Review by ryanwuer — APPROVED** [review]

**Pull request merged**